webapp-django/crashstats/settings/base.py

```diff
@@ -376,12 +346,260 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
+# web server thread.
+# Comment out to use memcache from settings/base.py
+CACHES = {
+    'default': {
+        # use django.core.cache.backends.memcached.MemcachedCache for prod
+        'BACKEND': config(
+            'CACHE_BACKEND',
+            'django.core.cache.backends.locmem.LocMemCache',
+        ),
+        'LOCATION': config(
+            'CACHE_LOCATiON',
+            'crashstats',
+        ),
+        'TIMEOUT': config('CACHE_TIMEOUT', 500),
+        'KEY_PREFIX': config('CACHE_KEY_PREFIX', 'crashstats'),
+    }
+}
+
+TIME_ZONE = config('TIME_ZONE', 'UTC')
+
+# FIXME should support URLs here!
+DATABASES = {
+    'default': {
+        # use django.db.backends.postgresql_psycopg for production
+        'ENGINE': config('DATABASE_ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': config('DATABASE_NAME', 'sqlite.crashstats.db'),
+        'USER': config('DATABASE_USER', ''),
+        'PASSWORD': config('DATABASE_PASSWORD', ''),
+        'HOST': config('DATABASE_HOST', ''),
+        'PORT': config('DATABASE_PORT', ''),
+        'OPTIONS': {
+        },
+        'TEST_CHARSET': 'utf8',
+        'TEST_COLLATION': 'utf8_general_ci',
+    },
+    # 'slave': {
+    #     ...
+    # },
+}
+
+
+# Uncomment this and set to all slave DBs in use on the site.
+SLAVE_DATABASES = config('SLAVE_DATABASES', None, cast=Csv)
+
+# TODO not sure how we can override this, should we bother?
+# Recipients of traceback emails and other notifications.
+ADMINS = (
+    # ('Your Name', 'your_email@domain.com'),
+)
+MANAGERS = ADMINS
+
+# Debugging displays nice error messages, but leaks memory. Set this to False
+# on all server instances and True only for development.
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', True, cast=bool)
+
+# Set this to True to make debugging AJAX requests easier; development-only!
+DEBUG_PROPAGATE_EXCEPTIONS = config(
+    'DEBUG_PROPAGATE_EXCEPTIONS',
+    True,
+    cast=bool
+)
+
+# By default compression is done in runtime, if you enable
+# offline compression, running the test suite will be 10 times faster
+# but you'll need to remember to first run:
+#     ./manage.py collectstatic --noinput
+#     ./manage.py compress --force --engine=jinja2
+# at least once every time any of the static files change.
+COMPRESS_OFFLINE = config('COMPRESS_OFFLINE', True, cast=bool)
+
+# Make this unique, and don't share it with anybody.  It cannot be blank.
+SECRET_KEY = config('SECRET_KEY', 'you must change this')
+
+# Uncomment these to activate and customize Celery:
+# required to activate celeryd
+CELERY_ALWAYS_EAGER = config('CELERY_ALWAYS_EAGER', False, cast=bool)
+BROKER_HOST = config('BROKER_HOST', 'localhost')
+BROKER_PORT = config('BROKER_PORT', 5672, cast=int)
+BROKER_USER = config('BROKER_USER', 'playdoh')
+BROKER_PASSWORD = config('BROKER_PASSWORD', 'playdoh')
+BROKER_VHOST = config('BROKER_VHOST', 'playdoh')
+CELERY_RESULT_BACKEND = config('CELERY_RESULT_BACKEND', 'amqp')
```

Comment:
I would just remove these ancient playdoh artifacts. If we one day start using celery we probably will want to start from scratch kinda.